webrtc.html

```diff
@@ -11573,9 +11573,89 @@ <h2 id="sec.idp-error-handling">IdP Error Handling</h2>
         <code>RTCError</code> constructor to create and initialize
         the subclass instance with an [[\ErrorData]] internal slot.</p>
         <section>
-      <h2>TODO errorDetailEnum</h2>
-      <p>TODO - make a enum for all the error details? Is the enum OK for
-      extending?<p>
+          <h2><dfn>RTCErrorDetailType</dfn> Enum</h2>
+          <div>
+          <pre class="idl">enum RTCErrorDetailType {
+              "data-channel-failure",
+              "dtls-failure",
+              "idp-bad-script-failure",
+              "idp-execution-failure",
+              "idp-load-failure",
+              "idp-need-login",
+              "idp-timeout",
+              "idp-tls-failure",
+              "idp-token-expired",
+              "idp-token-invalid",
+              "parameters-error",
+              "sctp-failure",
```

Comment:
Some of these (sctp-failure, parameters-error, dtls-failure, data-channel-failure) aren't used yet. May want to at least make a note of that, so we don't accidentally end up with an enum value that's not used anywhere.